### PR TITLE
 CSEN-23 "VmRequest" classes were implemented 

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    request.addParameter(DOMAIN_ID, id)
+    this
+  }
+}
+
+object VmCreateRequest {
+  /**
+    * Class is responsible for providing vm creation settings.
+    *
+    * @param serviceOfferingId the UUID of the service offering for the virtual machine
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:62]: what does it mean?

Comment (placccebo) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:62]: for -> of

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    * @param templateId the UUID of the template for the virtual machine
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:63]: for -> of

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    * @param zoneId the UUID of availability zone for the virtual machine
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:64]: for -> of

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:62]: Service Offering is a CloudStack entity - http://cloudstack.apache.org/api/apidocs-4.9/apis/listServiceOfferings.html

Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:62]: "for" is using in documentation - http://cloudstack.apache.org/api/apidocs-4.9/apis/deployVirtualMachine.html

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:63]: too

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala (hunk 3 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:64]: too

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala (hunk 1 above)
Comment (placccebo) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:62]: ok

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala (hunk 2 above)
Comment (placccebo) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:63]: ok

File: src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala (hunk 3 above)
Comment (placccebo) [reply src/main/scala/com/bwsw/cloudstack/entities/requests/VmCreateRequest.scala:64]: ok

**Review by placccebo — APPROVED** [review]

**Pull request merged**